test/unit/services/platform-test.js

```diff
@@ -74,4 +75,27 @@ describe('Platform', function() {
 
   });
 
+  describe('isVirtualizationEnabled', function(){
+
+    it('should able to check for mac', function() {
+      let stub = sandbox.stub(Platform, 'getOS').returns('darwin');
+      Platform.isVirtualizationEnabled();
+      expect(stub).to.have.been.calledOnce;
+    });
+
+    it('should able to check for linux', function() {
+      let stub = sandbox.stub(Platform, 'getOS').returns('linux');
+      Platform.isVirtualizationEnabled();
+      expect(stub).to.have.been.calledOnce;
+    });
+
+    it('should able to check for windows', function() {
+      let stub = sandbox.stub(Platform, 'getOS').returns('win32');
+      let stubs = sandbox.stub(child_process, 'spawn').yields(undefined, 'stdout', 'stderr');
+      Platform.isVirtualizationEnabled();
+      expect(stub).to.have.been.calledOnce;
```

Comment:
instead you should add tests 
``` JavaScript
describe('on windows', function(){
  it('should return promise resolved to true if powershelgl script returns `True` in stdout', function() {
    let stub = sandbox.stub(Platform, 'getOS').returns('win32');
    let stubs = sandbox.stub(child_process, 'spawn').yields(undefined, 'True', undefined);
    expect(Platform.isVirtualizationEnabled()).to.be.true;
  });
  it('should return promise resolved to true if powershelgl script returns `False` in stdout', function() {
    let stub = sandbox.stub(Platform, 'getOS').returns('win32');
    let stubs = sandbox.stub(child_process, 'spawn').yields(undefined, 'False', undefined);
    expect(Platform.isVirtualizationEnabled()).to.be.false;
  });
  it('should return promise resolved to undefined if powershelgl script returns nothing in stdout', function() {
    let stub = sandbox.stub(Platform, 'getOS').returns('win32');
    let stubs = sandbox.stub(child_process, 'spawn').yields(undefined, '', undefined);
    expect(Platform.isVirtualizationEnabled()).to.be.undefined;
  });
});
```
Something like this would do produce test report like this:
``` text
isVirtualizationEnabled
  on mac
    should return true if virtualization check shell script returns true
    should return false if virtualization check shell script returns false
    should return undefined if virtualization check shell script returns nothing
  on linux
    should return true if virtualization check shell script returns true
    should return false if virtualization check shell script returns false
    should return undefined if virtualization check shell script returns nothing
  on windows
    should return promise resolved to true if powershelgl script returns `True` in stdout
    should return promise resolved to true if powershelgl script returns `False` in stdout
    should return promise resolved to undefined if powershelgl script returns nothing in stdout
```